cypress/integration/unit_tests/click_feature_test.js

```diff
@@ -1,90 +1,201 @@
-       */
-      setPopup(popup) {
-        this.popup = popup;
-      }
-    }
+  let features;
+  let scoredFeatures;
+  let map;
 
-    cy.stub(HighlightFeatures, 'highlightFeatures', (features, map) => {
-      if (features.length === 0) {
-        HighlightFeatures.currentFeatures.clear();
-      } else {
-        HighlightFeatures.currentFeatures.set(
-            0, new HighlightFeatures.CurrentFeatureValue());
-      }
-    });
-
-    tableApi = {
-      setSelection: (selection) => {},
-    };
-    mockTable = Cypress.sinon.mock(tableApi);
-    HighlightFeatures.currentFeatures.clear();
-    const featureProperties = {'GEOID': 0};
-    const feature = ee.Feature(null, featureProperties);
-    const featureCollection = ee.FeatureCollection([feature]);
-    cy.stub(ee, 'FeatureCollection')
-        .withArgs('mockAsset')
-        .returns(featureCollection);
-    cy.stub(featureCollection, 'filterBounds').returns(featureCollection);
-    cy.stub(featureCollection, 'first').returns(feature);
-    cy.stub(feature, 'evaluate').callsFake((callb) => callb({
-                                             'type': 'Feature',
-                                             'geometry':
-                                                 {coordinates: [[[0, 0]]]},
-                                             'properties': featureProperties,
-                                           }));
+  before(() => {
+    const feature1 = createFeatureFromCorners(...feature1Corners)
+                         .set(blockGroupTag, 'my block group');
+    const feature2 = createFeatureFromCorners(...feature2Corners)
+                         .set(blockGroupTag, 'another group');
+    const offMapFeature = createFeatureFromCorners(10, 10, 20, 20);
+    const otherFeature = createFeatureFromCorners(-11, -11, -10, -10);
+    features =
+        ee.FeatureCollection([feature1, feature2, offMapFeature, otherFeature]);
+    scoredFeatures = ee.FeatureCollection([
+      feature1.set(scoreTag, 1),
+      feature2.set(scoreTag, 3),
+      offMapFeature.set(scoreTag, 2),
+      otherFeature.set(scoreTag, 0),
+    ]);
   });
 
-  it('clicks on a block group in the list', () => {
-    const tableData = [tableHeadings, [0, 99, 0.46, 0.52]];
-    mockTable.expects('setSelection').once().withArgs([{row: 0, column: null}]);
+  beforeEach(() => currentFeatures.clear());
 
-    clickFeature(0, 0, null, 'mockAsset', tableApi, tableData);
+  /**
+   * Sets up map and table by calling real setup functions, but with our toy
+   * EE features.
+   * @return {Cypress.Chainable}
+   */
+  function setUpPage() {
+    const loadingFinishedPromise =
+        new Promise((resolve) => loading.loadingElementFinished = (id) => {
+          if (id === tableContainerId) resolve();
+        });
+    createGoogleMap().then((mapResult) => map = mapResult);
+    cy.document().then((doc) => {
+      // Lightly fake out prod document access.
+      cy.stub(document, 'getElementById')
```

Comment:
we should probably standardize this eventually. We've been doing different things. maybe add a todo?